backend/composer/services/cs_ingestion/neurondm_script.py

```diff
@@ -31,55 +30,35 @@ def lrdf(neuron, predicate):
-            # XXX I strongly reccoment renaming this to SENSORY-T so that the
-            # short forms are harder to confuse A-T and S-T
-            [dict(loc=l, type='AFFERENT-T') for l in lpes(n, ilxtr.hasAxonSensorySubcellularElementIn)]
-        ),
-        path = (  # TODO pull ordering from partial orders (not implemented in core atm)
-            [dict(loc=l, type='AXON') for l in lpes(n, ilxtr.hasAxonLocatedIn)] +
-            # XXX dendrites don't really ... via ... they are all both terminal and via at the same time ...
-            [dict(loc=l, type='DENDRITE') for l in lpes(n, ilxtr.hasDendriteLocatedIn)]
-        ),
-        #laterality = lpes(n, ilxtr.hasLaterality),  # left/rigth tricky ?
-        #projection_laterality = lpes(n, ilxtr.???),  # axon located in contra ?
-        species =            lpes(n, ilxtr.hasInstanceInTaxon),
-        sex =                lpes(n, ilxtr.hasBiologicalSex),
-        circuit_type =       lpes(n, ilxtr.hasCircuitRolePhenotype),
-        phenotype =          lpes(n, ilxtr.hasAnatomicalSystemPhenotype),  # current meaning of composer phenotype
-        anatomical_system =  lpes(n, ilxtr.hasAnatomicalSystemPhenotype),
-        # there are a number of dimensions that we aren't converting right now
-        dont_know_fcrp =     lpes(n, ilxtr.hasFunctionalCircuitRolePhenotype),
-        other_phenotype = (  lpes(n, ilxtr.hasPhenotype)
-                           + lpes(n, ilxtr.hasMolecularPhenotype)
-                           + lpes(n, ilxtr.hasProjectionPhenotype)),
-        forward_connection = lpes(n, ilxtr.hasForwardConnectionPhenotype),
-
-        # direct references from individual individual neurons
-        provenance =      lrdf(n, ilxtr.literatureCitation),
-        sentence_number = lrdf(n, ilxtr.sentenceNumber),
-        note_alert =      lrdf(n, ilxtr.alertNote),
-        # XXX provenance from ApiNATOMY models as a whole is not ingested
-        # right now because composer lacks support for 1:n from neuron to
-        # prov, (or rather lacks prov collections) and because it attaches
-        # prov to the sentece, which does not exist for all neurons
-
-        # TODO more ...
-        # notes = ?
-
-        # for _ignore, hasClassificationPhenotype is used for ApiNATOMY
-        # unlikely to be encountered for real neurons any time soon
-        _ignore = lpes(n, ilxtr.hasClassificationPhenotype),  # used to ensure we account for all phenotypes
+        id=str(n.id_),
+        label=str(n.origLabel),
+        origins=origins,
+        destinations=destinations,
+        vias=vias,
+        species=lpes(n, ilxtr.hasInstanceInTaxon),
+        sex=lpes(n, ilxtr.hasBiologicalSex),
+        circuit_type=lpes(n, ilxtr.hasCircuitRolePhenotype),
+        circuit_role=lpes(n, ilxtr.hasFunctionalCircuitRolePhenotype),
+        phenotype=lpes(n, ilxtr.hasAnatomicalSystemPhenotype),
+        #classification_phenotype=lpes(n, ilxtr.hasClassificationPhenotype),
```

Comment:
I noticed that we were not using this in composer so I commented it out for now. Please let me know if we should be using this somewhere in composer

Reply:
@afonsobspinto I'm not aware that we use this so we can remove it